bundles/org.openhab.binding.nanoleaf/src/main/resources/ESH-INF/i18n/nanoleaf_de.properties

```diff
@@ -1,9 +1,9 @@
 binding.nanoleaf.name = Nanoleaf Binding
-binding.nanoleaf.description = Binding f�r die Integration des Nanoleaf Light Panels
+binding.nanoleaf.description = Binding f�r die Integration des Nanoleaf Light Panels
```

Comment:
Umlaute seem to be screwed-up again

Reply:
My bad while merging :-(
What encoding should the file have? (UTF-8)

Reply:
I was finally able to see and fix the issue in the Visual Code editor.